xdsl/dialects/arith.py

```diff
@@ -81,6 +82,13 @@ def from_int_constant(val: Union[int, Attribute],
             result_types=[typ],
             attributes={"value": IntegerAttr.from_params(val, typ)})
 
+    @staticmethod
+    def from_anyfloat_constant(val: Union[float, Attribute],
```

Comment:
Can we rename it to `from_float_constant`?
Actually, should we rename them to `from_int` and `from_float`?

Reply:
Agree, changing